### PR TITLE
Integrate sort query to search page

### PR DESCRIPTION
**Pull request by DLi7077**
Added a component in search headers for handling sort keys and sort direction
Desktop:
image

Mobile:
image

### CODE REVIEW

File: client/src/pages/Search/SearchHeader/PageNavigation.js [hunk 1]
```diff
@@ -35,7 +35,7 @@ export default function PageNavigation({
         <ChevronLeftIcon />
       </IconButton>
       <div>
-        Found {lowerBound}-{upperBound} of {total_count}
+        Showing {lowerBound}-{upperBound} of {total_count}
```
Comment (DLi7077) [inline client/src/pages/Search/SearchHeader/PageNavigation.js:38]: figured that "Showing" makes more sense

Comment (Remos96) [reply client/src/pages/Search/SearchHeader/PageNavigation.js:38]: Agreed

File: server/src/services/restaurant.ts [hunk 2]
```diff
@@ -237,7 +237,11 @@ function generateSortQuery(
   // Sort by distance
   if (sortBy === SORT_KEY.DISTANCE) {
     // query defaulted to sort by distance
-    return [];
+    const sortByDistance = {
+      $sort: { "restaurant.distance": sortDirectionValue },
```
[the remark this answers is not in the document]
Comment (Remos96) [reply server/src/services/restaurant.ts:241]: Doubt many would, but still nice to have

**Review by Remos96 — APPROVED** [review]
Lookin sweetsicle, great work

**Pull request merged**